### ISSUE TITLE
[Possible BUG]同じ分類記号が大量に生成される

### ISSUE THREAD

**Issue by is984** [next-l/enju_leaf#1769]
## 問題(Problem)
同じ分類記号が大量に登録される
<img width="538" alt="image" src="https://user-images.githubusercontent.com/22042489/206080178-d2fdfb0a-76dd-426b-bb3d-32f95eab1d56.png">
※当該環境では913.6が85個ある(NDC10と9合わせて)

## 再現方法 ( Way to Reproduce)
不明。書籍に分類を設定するとおそらく新しい分類が無制限に作成される

## 期待される動作 ( Expected result )
同一の分類記号が登録されない。
-> 例 : NDC10と9のそれぞれに1つずつ913.6があることが正常な動作と期待する。
## 備考(Note)
おそらく[これ](https://github.com/next-l/enju_leaf/issues/758)が解決されないまま1.4まで残っているものと推定されます。

**Comment by nabeta** [comment]
@is984 はい、https://github.com/next-l/enju_leaf/issues/758 の件になります。これはデータの持ち方を変えて対応することを考えています。
